src/Aspire.Hosting.Nats/NatsBuilderExtensions.cs

```diff
@@ -30,19 +31,28 @@ public static IResourceBuilder<NatsServerResource> AddNats(this IDistributedAppl
     /// <summary>
     /// Adds JetStream support to the NATS server resource.
     /// </summary>
-    /// <param name="builder">NATS resource builder.</param>
-    /// <param name="srcMountPath">Optional mount path providing persistence between restarts.</param>
-    /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
-    public static IResourceBuilder<NatsServerResource> WithJetStream(this IResourceBuilder<NatsServerResource> builder, string? srcMountPath = null)
-    {
-        var args = new List<string> { "-js" };
-        if (srcMountPath != null)
-        {
-            args.Add("-sd");
-            args.Add("/data");
-            builder.WithBindMount(srcMountPath, "/data");
-        }
+    /// <param name="builder">The resource builder.</param>
+    /// <returns>The <see cref="IResourceBuilder{T}"/>.</returns>
+    public static IResourceBuilder<NatsServerResource> WithJetStream(this IResourceBuilder<NatsServerResource> builder)
+        => builder.WithArgs("-js", "-sd", "/var/lib/nats");
```

Comment:
What happens if you call this but you don't call `WithDataVolume`

Reply:
Seems like a good test to add.

Reply:
I think data just stays on the ephemeral container storage. Could you give me some ideas about the test. Happy to add them too.

Reply:
I think we should split out `WithJetStream` and `WithDataBindMount` and `WithDataVolume`. What I mean by that is that `WithJetStream` won't add the `-sd [path]` argument. Lets make that someting that people can do explicitly.

People might want to enable the JetStream feature set but don't care about persistence.

Reply:
where would you keep path value to be used by `WithDataBindMount` and `WithDataVolume`? Just not sure how to implement that.